src/types/query.d.ts

```diff
@@ -14,6 +14,8 @@ export module query {
   export function Object(fields: ExprArg): Expr;
   export function Lambda(f: Lambda): Expr;
   export function Lambda(var_name: ExprArg, expr: ExprArg): Expr;
+  export function Call(ref: ExprArg, ...args: ExprArg[]): Expr;
+  export function Query(lambda: ExprArg): Expr;
```

Comment:
Isn't `lambda` supposed to be of type `Lambda`?

Reply:
not necessarily, I could be a return of `Lambda()` function as well, so I will add a overload with `Lambda` type